File: app/src/main/java/com/example/myanimedata/api/ProducerResult.java
package com.example.myanimedata.api;

import com.google.gson.annotations.SerializedName;

public class ProducerResult {
    @SerializedName("mal_id")
    private final int id;

    @SerializedName("name")
    private final String name;

    @SerializedName("url")
    private final String url;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getUrl() {
        return url;
    }

    public ProducerResult(int id, String name, String url) {
        this.id = id;
        this.name = name;
        this.url = url;
    }
}

File: app/src/main/java/com/example/myanimedata/api/ProducerResult.kt
package com.example.myanimedata.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class ProducerResult(
    @field:SerializedName("mal_id") val id: Int,
    @field:SerializedName("name") val name: String,
    @field:SerializedName("url") val url: String
) : Parcelable